server/enterprise/message_export/actiance_export/actiance_export.go

```diff
@@ -224,6 +234,63 @@ func ActianceExport(rctx request.CTX, p Params) (shared.RunExportResults, error)
 	return results, err
 }
 
+func getPostExport(posts []*model.MessageExport, i int, results shared.RunExportResults) (PostExport, shared.RunExportResults) {
+	// We have three "kinds" of posts:
+	// (using "1" and "2" for simplicity)
+	// - created:                         Id = new,  CreateAt = 1,    UpdateAt = 1, DeleteAt = 0
+	// - deleted:                         Id = orig, CreateAt = orig, UpdateAt = 2, DeleteAt = 2, props: deleteBy
+	// - edited: old post gets "created": Id = new,  CreateAt = 1,    UpdateAt = 2, DeleteAt = 2, originalId: orig
+	//           existing post modified:  Id = orig, CreateAt = 1,    UpdateAt = 2, DeleteAt = 0
+	//
+	// We also have other ways for a post to be updated:
+	//  - a root post in a thread is replied to, when a reply is edited, or (as of 10.2) when a reply is deleted
+
+	post := posts[i]
+	if isEditedOriginalMsg(post) {
```

Comment:
I'm not 100% sure, since it's from before my time. But it seems like the `DeleteByProp` is for messages that are actually deleted. If there is no `DeleteByProp` then it's something else (it can identify if this is an edit history message, but I'm not sure if it helps identify another type of message). In the case of the `isEditedOriginalMsg` check, we don't even need to check the `DeleteByProp` because the `post.PostOriginalId` is enough to know it's an edited original msg.